scripts/openzfs-merge.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+#
+# This script tries to merge OpenZFS commits to ZoL.
+#
+# Instruction:
+# 
+# Repository setup must be similar with openzfs-tracking.sh
+# requirements.
+#
+# Repository setup for valid compilation check:
+#	sh autogen.sh
+#	./configure --enable-debug
+#
+# mandatory git settings:
+#	[merge]
+#   	renameLimit = 999999
+#	[user]
+#       email = mail@gmelikov.ru
+#       name = George Melikov
+#
+# Copyright (c) 2016 George Melikov. All rights reserved.
+#
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+REPOSITORY_PATH='.'
+TMP_FILE='/tmp/gittmpmessage.txt'
```

Comment:
I think you are using a file because it is easier to work with? If so, then this fine.